.github/workflows/notify_updates_on_github.yaml

```
@@ -0,0 +1,83 @@
+      - edited
+      - synchronize
+      - reopened
+      - closed
+
+jobs:
+  notify-slack:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine Event Info
+        id: info
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)